CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changes by Version
 ==================
 
+0.13.0 (2020-10-21)
```

Comment:
Could you change it to 22? I'll double-check the release notes tomorrow and merge/release.

Reply:
sure. It was done. Thanks.